META-INF/plugin.xml

```diff
@@ -9,5 +9,5 @@
   <key>EC-DslDeploy</key>
   <label>EC-DslDeploy</label>
   <pluginWizardBased>true</pluginWizardBased>
-  <version>1.0.0</version>
+  <version>1.0.1</version>
```

Comment:
@lrochette  since we are in development phase and have not yet released version 1.0.0, we can keep the version number the same. Just increment your build counter.

Reply:
@lrochette since we are in dev mode while iterating over this, lets keep the plugin version at 1.0.0.

Reply:
just discussed with @lrochette . We need to use version 1.0.1 as 1.0.0 was provided as an early release to some users.